commit 4473e3c780ec687112f22860d47006dd1f95b920
Author: Venkat Subbiah <venkat.subbiah@cavium.com>
Date:   Tue Apr 10 15:25:45 2012 -0700

    procps changes to get it to compile in SDK environment.

diff --git a/procps-3.2.7/Makefile b/procps-3.2.7/Makefile
index 9de3296..8b2846b 100644
--- a/procps-3.2.7/Makefile
+++ b/procps-3.2.7/Makefile
@@ -174,7 +174,7 @@ INSTALL := $(BINFILES) $(MANFILES)
 # want this rule first, use := on ALL, and ALL not filled in yet
 all: do_all
 
--include */module.mk
+-include proc/module.mk ps/module.mk 
 
 do_all:    $(ALL)
 
@@ -222,14 +222,14 @@ clean:
 ###### install
 
 $(BINFILES) : all
-	$(install) --mode a=rx $(notdir $@) $@
+	cp --remove-destination  $(notdir $@) $@
 
 $(MANFILES) : all
-	$(install) --mode a=r $(notdir $@) $@
+	#$(install) --mode a=r $(notdir $@) $@
 
 install: $(filter-out $(SKIP) $(addprefix $(DESTDIR),$(SKIP)),$(INSTALL))
-	cd $(usr/bin) && $(ln_f) skill snice
-	cd $(usr/proc/bin) && $(ln_f) pgrep pkill
+	#cd $(usr/bin) && $(ln_f) skill snice
+	#cd $(usr/proc/bin) && $(ln_f) pgrep pkill
 
 ############ prog.c --> prog.o
 
diff --git a/procps-3.2.7/proc/module.mk b/procps-3.2.7/proc/module.mk
index af20334..59e4df7 100644
--- a/procps-3.2.7/proc/module.mk
+++ b/procps-3.2.7/proc/module.mk
@@ -96,7 +96,7 @@ endif
 #################### install rules ###########################
 
 $(lib)$(SOFILE) : proc/$(SONAME)
-	$(install) --mode a=rx $< $@
+	cp --preserve --remove-destination $< $@
 
 ifneq ($(SOLINK),$(SOFILE))
 .PHONY: $(lib)$(SOLINK)
@@ -115,7 +115,7 @@ ldconfig : $(lib)$(SONAME) $(lib)$(SOLINK)
 	$(ldconfig)
 
 $(usr/lib)$(ANAME) : proc/$(ANAME)
-	$(install) --mode a=r $< $@
+	cp -remove-destination $< $@
 
 # Junk anyway... supposed to go in /usr/include/$(NAME)
 #INSTALL += $(addprefix $(include),$(HDRFILES))
diff --git a/procps-3.2.7/ps/module.mk b/procps-3.2.7/ps/module.mk
index 2902a3a..2ee02fb 100755
--- a/procps-3.2.7/ps/module.mk
+++ b/procps-3.2.7/ps/module.mk
@@ -33,8 +33,8 @@ ps/stacktrace.o: ps/stacktrace.c
 
 
 $(bin)ps: ps/ps
-	$(install) --mode a=rx $< $@
+	cp --remove-destination $< $@
 
 $(man1)ps.1 : ps/ps.1
-	$(install) --mode a=r $< $@
-	-rm -f $(DESTDIR)/var/catman/cat1/ps.1.gz $(DESTDIR)/var/man/cat1/ps.1.gz
+	#cp --remove-destination $< $@
+	#-rm -f $(DESTDIR)/var/catman/cat1/ps.1.gz $(DESTDIR)/var/man/cat1/ps.1.gz
